src/votekit/ballot_generator.py

```diff
@@ -2106,19 +2106,29 @@ class Clustered_DSpatial(BallotGenerator):
     def __init__(
         self,
         candidates: list[str],
-        voter_dist: Callable[..., np.ndarray] = np.random.uniform,
+        voter_dist: Callable[..., np.ndarray] = np.random.normal,
         voter_params: Optional[Dict[str, Any]] = None,
         candidate_dist: Callable[..., np.ndarray] = np.random.uniform,
         candidate_params: Optional[Dict[str, Any]] = None,
         distance: Callable[[np.ndarray, np.ndarray], float] = euclidean_dist,
     ):
         super().__init__(candidates=candidates)
         self.candidate_dist = candidate_dist
+        self.voter_dist = voter_dist
 
         if voter_params is None:
             # default params used for np.random.normal
-            self.voter_params = {"std": np.array(1.0), "size": np.array(2.0)}
+            self.voter_params = {"loc": 0, "std": np.array(1.0), "size": np.array(2.0)}
         else:
+            # Trying to ensure that the input distribution's mean is parameterized.
+            # This allows me to center it on the candidates themselves later
+            try:
+                voter_params['loc'] = 0
+                self.voter_dist(**voter_params)
```

Comment:
I tried to fix Clustered_DSpatial so that it can take any general distribution for the voters. The only requirement it should satisfy is that the distribution has a 'loc' parameter specifying the mean of the distribution. Like in np.random.normal(loc = 0). Let me know if you have any thoughts or suggestions for how to test the input distribution for that.